### ISSUE TITLE
Keyboard Navigation Fails At Typeahead in Editable Cell.

### ISSUE THREAD

**Issue by totalard** [angular-ui/ui-grid#5714]
My Editable Cell Working Perfect.. And Cell Nav Function is also Implemented. When dropdown is activated in Typeahead, I have to press Downbutton to navigate through Dropdown of typeahead. But It changes the cell to the bottom.. The Navigation Not working at all. The Issue detected only in Typeahead right now. 

**Kindly Inform me how i can disable down and upward navigation when navigating in a certain column.**

**Comment by totalard** [comment]
I have Got these from Docs

> When combined with the editable feature, the left-right arrow keys will be subsumed when in "deep edit" mode, allowing you to move around within the text you're editing. The tab key and shift-tab keys continue to function.


I want to Subsume up and Down too in certain column at Edit mode,, That is the SUMMARY

**Comment by totalard** [comment]
No One Here!!!

**Comment by mportuga** [comment]
@totalard I will take a look at this tomorrow and reply to you as soon as I can.

**Comment by csimpi** [comment]
same here
